feature/client-charge/src/commonMain/kotlin/org/mifos/mobile/feature/charge/charges/ClientChargeScreen.kt

```diff
@@ -95,23 +125,59 @@ internal fun ClientChargeScreen(
     )
 }
 
-/**
- * Composable function that displays the Client Charges Screen.
- *
- * @param state State of the screen.
- * @param modifier Modifier to be applied to the layout.
- * @param onAction A lambda function that is called when the user performs an action.
- */
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 private fun ClientChargeScreen(
     state: ClientChargeState,
     modifier: Modifier = Modifier,
     onAction: (ClientChargeAction) -> Unit,
 ) {
-    MifosElevatedScaffold(
-        topBarTitle = stringResource(state.topBarTitleResId),
-        onNavigateBack = { onAction(ClientChargeAction.OnNavigate) },
+    val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+
+    Scaffold(
         modifier = modifier,
+        containerColor = Color.White,
+        topBar = {
+            TopAppBar(
+                title = {
+                    Column {
+                        Text(
+                            text = stringResource(state.topBarTitleResId),
+                            style = MaterialTheme.typography.titleMedium.copy(
```

Comment:
Replace `MaterialTheme` with `KptTheme` everywhere. Before that update your branch locally with the latest changes made in the `dev`
```suggestion
                            style = KptTheme.typography.titleMedium.copy(
```